docs/validators/system-requirements.md

```diff
@@ -30,12 +30,28 @@ Nodes are computers running the MultiversX software, so they contribute to the M
 :::
 
 :::tip
-1. Support for ARM processors (e.g. for Raspberry Pi) will come in the future, pending third-party issues.
-2. We are promoting using processors that support the `fma` or `fma3` instruction set since it is widely used by our VM. Displaying the available CPU instruction set can be done using the Linux shell command `sudo lshw` or `lscpu`
+We are promoting using processors that support the `fma` or `fma3` instruction set since it is widely used by our VM. Displaying the available CPU instruction set can be done using the Linux shell command `sudo lshw` or `lscpu`
 :::
 
 [comment]: # (mx-context-auto)
 
+## **ARM Architecture Support**
+
+Processors with ARM architecture are now supported.
```

Comment:
maybe add something like: we are supporting ARM processors on mainnet only starting from epoch 1265 (related to this release https://github.com/multiversx/mx-chain-mainnet-config/releases/tag/v1.6.7.0) 
It is important to state this because syncing from genesis to epoch 1265 is not possible on ARM processors.

Reply:
Done.